tests/ops/fixtures/saas/zendesk_fixtures.py

```diff
@@ -107,6 +110,8 @@ def zendesk_create_erasure_data(
     zendesk_connection_config: ConnectionConfig, zendesk_erasure_identity_email: str
 ) -> None:
 
+    time.sleep(60)
```

Comment:
We need to wait performing the access request or else we will get a rate limit exception when we try to create the erasure data. More robust solution coming up in a future ticket.